packages/react-shortcuts/README.md

```diff
@@ -0,0 +1,156 @@
+# `@shopify/react-shortcuts`
+
+![bundle size badge](https://img.shields.io/bundlephobia/minzip/@shopify/react-shortcuts.svg)
+
+Declaratively and efficiently match shortcut combinations in your react application.
```

Comment:
This describes why to use the package, or what its benefit is, but not what *it* is. Perhaps we should establish a clear convention here too. I like the following sentence structure:

> Declarative and efficient library for matching shortcut combinations in React applications.